app/controllers/issues_controller.rb

```diff
@@ -22,7 +23,7 @@ def new
   end
 
   def show
-    @issue = Issue.find(params[:id])
+    @voted = @issue.votes.where(user_id: current_user.id)
     load_relevant_issues
     redirect_back(fallback_location: root_path) unless \
                                                 @issue.can_read?(current_user)
```

Comment:
ok, my comment is not valid because i confused the objects, I aploagize for that. Another thing: could you move
```ruby
redirect_back(fallback_location: root_path) unless \
                                                 @issue.can_read?(current_user)
```

into ```init_issue``` method?